src/Uno.Extensions.Navigation.Generators/Uno.Extensions.Navigation.Generators.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+		<IsPackable>false</IsPackable>
+
+		<!--
+			Defines that this project output is a tool of the given package ID,
+			so nuget override will copy it in the 'tools' directory of that package instead of the 'lib' of the current package folder.
```

Comment:
```suggestion
			so nuget override will copy it in the 'analyzers' directory of that package instead of the 'lib' of the current package folder.
```